### DISCUSSION TITLE
Winning Percentage/Chance to Win

### DISCUSSION THREAD

**Discussion by MrDGer** [cwendt94/espn-api#367] [ESPN API] [answered]
Is there already a function or call for the live data that ESPN refers to as the "Chance to Win" on their FantasyCast page?

**Comment by dtcarls** [comment, marked as answer]
I would love this feature. I've looked into this before, and as far as I can tell it is handled in the javascript of the FantasyCast page. Unfortunately the js minified so I was unable to locate where the logic was. 

It's located in the "totalPerc" div, and I believe the logic is located somewhere in fantasycast.js if someone is better at deciphering minified javascript than myself.

**Reply by MrDGer** [reply]
Thanks for the response!  Definitely beyond my capabilities, but good to know I wasn't just missing it somewhere simple.

**Reply by cwendt94** [reply]
Adding on to @dtcarls answer, I believe for the FantasyCast page ESPN is using web sockets instead of their traditional API's to get the data. Its a lot harder to hook into their web sockets and process the data.

**Reply by MrDGer** [reply]
Understood! No problem at all, thanks for the response!
